.github/workflows/integration.yml

```diff
@@ -186,6 +186,7 @@ jobs:
   postgresql:
     name: "PostgreSQL Integration Tests"
     runs-on: ubuntu-latest
+    env:
```

Comment:
Does this parse properly? I don't see the CI run in this PR

Reply:
Fixed this up and both drivers are failling as expected. I also removed the LD_PRELOAD piece, which I think is only required when executing the libraries from Python (assuming Python itself wasn't built with sanitizer support)